pocs/focuser/birger.py

```diff
@@ -112,7 +130,6 @@ def move_to(self, position):
         Does not do any checking of the requested position but will warn if the lens reports hitting a stop.
         Returns the actual position moved to in lens encoder units.
         """
-        self.logger.debug('Moving focus to {} encoder units'.format(position))
```

Comment:
I try to keep `INFO` for basic information you see when running a session. So if you load a `POCS` instance and initialize it, you get:

```
In [2]: pocs = POCS(simulator=['camera', 'weather', 'night'])
> 2017-02-10 21:29:31 ganymede panoptes[11144] INFO ******************************** Starting POCS *********************************
> 2017-02-10 21:29:31 ganymede panoptes[11144] INFO Initializing PANOPTES unit
> 2017-02-10 21:29:31 ganymede panoptes[11144] INFO Loading state table: simple_state_table
> 2017-02-10 21:29:31 ganymede panoptes[11144] INFO 	Initializing observatory
> 2017-02-10 21:29:31 ganymede panoptes[11144] INFO 		 Setting up location
> 2017-02-10 21:29:31 ganymede panoptes[11144] INFO 		 Setting up mount
> 2017-02-10 21:29:31 ganymede panoptes[11144] INFO Loading mount commands file: /storage/panoptes/POCS/resources/mounts/bisque.yaml
> 2017-02-10 21:29:31 ganymede panoptes[11144] INFO 		 Setting up cameras
> 2017-02-10 21:29:31 ganymede panoptes[11144] INFO 		 Setting up scheduler
> 2017-02-10 21:29:32 ganymede panoptes[11144] INFO 	 Observatory initialized
> 2017-02-10 21:29:34 ganymede panoptes[11144] INFO Welcome GeeBot!

In [3]: pocs.initialize()
> 2017-02-10 21:29:34 ganymede panoptes[11144] INFO Connecting to mount
> 2017-02-10 21:29:34 ganymede panoptes[11144] INFO Mount connected
> 2017-02-10 21:29:34 ganymede panoptes[11144] INFO Initializing pocs.mount.bisque mount
> 2017-02-10 21:29:34 ganymede panoptes[11144] WARNING TheSkyX requires location to be set in the application
> 2017-02-10 21:29:34 ganymede panoptes[11144] INFO Mount initialized: True
```

From experience of watching the logs while running, you don't really want much more than that. You get some basic info but it's pretty high level and is assuming that someone is actually reading (most) of it while in use.